### PR TITLE
Create standalone builds.

### LINKED ISSUE #2: Remove lodash dependency?
Hey there!

What's your JS compilation and browser support look like these days? If I submit a PR removing the lodash dependency, since ES5 is pretty universal and easy to polyfill anyway, could that be merged here without breaking anything?

### PR DESCRIPTION
**Pull request by donmccurdy**
Fixes #2.

This adds two builds – one ("lite") depends on lodash and weighs 8kb minified. The second includes all its dependencies, and weighs 11kb. I assume you'll want the first, but the second seems more user-friendly for NPM and Bower. Or maybe 3kb is a pointless optimization for an internal package, and we should get rid of the "lite" build?

Would have liked to use the `lodash.*` modules, but individually packaging the parts we use add +60kb. So I've cut corners and used dedicated NPM packages like `object-assign` instead, for the standalone build. +3kb vs +60kb for the dependencies seems like a win.

Finally, I removed Grunt, since it's overkill and you use Gulp anyway. There are a few basic tasks defined in package.json – `npm run dev`, `npm run dist`, `npm version [major | minor | patch]`, etc.

Sending as a PR to sanity-check this – are you OK with including `dist/tagselect.lite.js` from your submodule? (Assuming you don't just stick with v0.6.* indefinitely.)

If we want to merge this I'll add docs and bump to v1.0.0.

### CODE REVIEW

File: src/Autocomplete.js [hunk 1]
```diff
@@ -159,50 +163,17 @@ Autocomplete.prototype.onInput = function () {
...
 		e.preventDefault();
 		e.stopImmediatePropagation();
 		this.select(this.dd.querySelector('.active'));
 		this.hide();
-	} else if (_.contains(ARROW_KEYS, e.which)) {
+	} else if (ARROW_KEYS.indexOf(e.which) >= 0) {
 		e.preventDefault();
 		this.setActive(e.which === UP_KEY ? 'previous' : 'next');
-	} else if (_.contains(HIDE_KEYS, e.which)) {
+	} else if (HIDE_KEYS.indexOf(e.which)) {
```
Comment (dylanwenzlau) [inline src/Autocomplete.js]: did you mean to use `>= 0` here too?

**Comment by schnerd** [comment]
Looks good to me! Go ahead and make those docs / version bump changes and we can merge.

**Comment by donmccurdy** [comment]
Docs are updated! I can't bump the version in the PR (without manually doing all the git tagging afterward) but either of us can run `npm version major` to bump it once this is in.

After this PR we won't need to rebuild `dist/` in commits either – that happens automatically on version release, basically like a deploy.

**Comment by jdalton** [comment]
Do you know the size comparison (minified+gizpped) of using `lodash` + `lodash-webpack-plugin` vs. this route?

**Comment by jdalton** [comment]
For smaller builds the `lodash/xyz` should be used over `lodash.xyz`.

**Comment by donmccurdy** [comment]
@jdalton Thanks! The web pack plugin is a 2x improvement over `lodash/xyz` alone after gzipping.

Here are my numbers:

* **misc. modules + Browserify** – 11kb minified, 4kb gzipped
* **lodash/xyz + Browserify** – 67kb minified, 15kb gzipped
* **lodash/xyz + Webpack** – 21kb minified, 7kb gzipped

It's still not quite as compact as the hand-picked assortment of modules, but I'm probably OK with a 3kb hit to have the same code running in both builds. Opening PR #4.

**Pull request merged**